website/server/libs/taskManager.js

```diff
@@ -86,6 +86,11 @@ export async function createTasks (req, res, options = {}) {
     let taskType = taskData.type;
     let newTask = new Tasks[taskType](Tasks.Task.sanitize(taskData));
 
+    // Attempt to round priority
+    if (newTask.priority && Number.parseFloat(newTask.priority) !== NaN) {
```

Comment:
I don't think you can compare NaN with `===`, it looks like you have to use `Number.isNaN` https://developer.mozilla.org/it/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN